### PR TITLE
Add reference to MAPRG 111 lightning talk

### LINKED ISSUE #88: Mention "Comparative Latency Under Load Performance of Broadband CPE" in MAPRG at IETF 111?
Jason Livingood's presentation is at https://datatracker.ietf.org/doc/slides-111-maprg-comparative-latency-under-load-performance-of-broadband-cpe/ - mention experience with DOCSIS 3.1 AQM?

### PR DESCRIPTION
**Pull request by SpencerDawkins**
and provide context.

close #88

### CODE REVIEW

**Comment by GrumpyOldTroll** [comment]
This seems out of place here to me, but I can see why it's hard to find a spot for the AQM reference since we don't have anything else that talks about network latency.  Hmm, how badly do we think we need this?

**Comment by SpencerDawkins** [comment]
We like part of one sentence in this PR, that we think does belong in the document, but we aren't sure it belongs where it is in the document. 

> ~~The rest of this section describes how ABR works, but~~ one thing to keep in mind is that ABR is an application-level mechanism that can improve the user's experience, but cannot provide a better experience than the underlying network path can provide.

**Comment by SpencerDawkins** [comment]
@GrumpyOldTroll and @acbergen - I note that in #155, @zaheduzzaman asked about mentioning active testing in Section 5. 

Although the lightning talk @jlivingood gave was describing an active testing system Comcast had deployed to monitor their access networks, and we've agreed that the underlying network was a "black box" from the perspective of media providers, ISTM that over-the-top media providers could perform active testing as well, between their origin servers and the first downstream media receivers (whether that is a CDN or endpoint media players), in order to detect significant **changes** in performance for a known workflow. 

I plan to add a mention of active testing in Section 5 as part of this PR, referencing Jason's lightning talk, rather than trying to use the lightning talk in two parallel PRs.

**Comment by GrumpyOldTroll** [comment]
I agree this is possible in theory, but I'm not aware of any operational experience that's been collected on these techniques.

@zaheduzzaman, I don't suppose you have a reference to work that describes using active measurements to improve media streaming delivery?

File: draft-ietf-mops-streaming-opcons.md [hunk 1]
```diff
@@ -664,20 +664,25 @@ In some applications, optimizations are available to on-demand video that are no
 
 # Adaptive Encoding, Adaptive Delivery, and Measurement Collection {#sec-abr}
 
-## Overview
+This section describes one way of providing the best user experience possible over a given network path, but one thing to keep in mind is that application-level mechanisms cannot provide a better experience than the underlying network path can support.
```
[the remark this answers is not in the document]
Comment (SpencerDawkins) [reply draft-ietf-mops-streaming-opcons.md:667]: Agreed. It's in my next commit.

File: draft-ietf-mops-streaming-opcons.md [hunk 2]
```diff
@@ -664,20 +664,25 @@ In some applications, optimizations are available to on-demand video that are no
...
+Adaptive bitrate streaming ("ABR streaming", or simply "ABR") is a commonly used technique for dynamically adjusting the compression level and video quality of a stream to match bandwidth availability. When this goal is achieved, the media server will tend to send enough media that the media player does not "stall", without sending so much media that the media player cannot accept it, with the excess media being buffered along the network path, increasing latency and, if latency is not consistent over time, potentially presenting the user with a "choppy" listening and viewing experience.
+
+ABR uses an application-level response strategy in which the streaming client attempts to detect the available bandwidth of the network path by observing the successful application-layer download speed, and then choosing a bitrate for each of the video, audio, subtitles and metadata (among a limited number of available options for each type of media) that fits within that bandwidth, typically adjusting as changes in available bandwidth occur in the network or changes in capabilities occur during the playback (such as available memory, CPU, display size, etc.).
```
[the remark this answers is not in the document]
Comment (SpencerDawkins) [reply draft-ietf-mops-streaming-opcons.md:685]: I worked on this a bit - you're right, it was fabulously opaque. It's now (shown as bullets here, for clarity, but not in the doc itself): 

- ABR uses an application-level response strategy in which the streaming client attempts to detect the available bandwidth of the network path by first observing the successful application-layer download speed, and then, 
- given the available bandwidth, the client chooses a bitrate for each of the video, audio, subtitles and metadata (among a limited number of available options for each type of media) that fits within that bandwidth, typically adjusting as changes in available bandwidth occur in the network or changes in capabilities occur during the playback (such as available memory, CPU, display size, etc.).

File: draft-ietf-mops-streaming-opcons.md [hunk 3]
```diff
@@ -664,20 +664,25 @@ In some applications, optimizations are available to on-demand video that are no
...
 
-ABR is a sort of application-level response strategy in which the streaming client attempts to detect the available bandwidth of the network path by observing the successful application-layer download speed, then chooses a bitrate for each of the video, audio, subtitles and metadata (among the limited number of available options) that fits within that bandwidth, typically adjusting as changes in available bandwidth occur in the network or changes in capabilities occur during the playback (such as available memory, CPU, display size, etc.).
+Adaptive bitrate streaming ("ABR streaming", or simply "ABR") is a commonly used technique for dynamically adjusting the compression level and video quality of a stream to match bandwidth availability. When this goal is achieved, the media server will tend to send enough media that the media player does not "stall", without sending so much media that the media player cannot accept it, with the excess media being buffered along the network path, increasing latency and, if latency is not consistent over time, potentially presenting the user with a "choppy" listening and viewing experience.
```
[the remark this answers is not in the document]
Comment (SpencerDawkins) [reply draft-ietf-mops-streaming-opcons.md:683]: I went with "cut". I was trying to be excessively precise, and that wasn't a great plan in this document.

**Pull request merged**